mibidata/tests/test_tiff.py

```diff
@@ -248,6 +249,8 @@ def test_sort_channels_before_writing(self):
 
         tiff.write(self.filename, unordered_image)
         image = tiff.read(self.filename)
+        print(image.metadata())
+        print(self.image.metadata())
```

Comment:
Here again, can we remove the printouts?

Reply:
Will do.